bamoe-ddl/postgresql/jobs-service.sql

```diff
@@ -0,0 +1,74 @@
+--
+-- Jobs service uses below tables
+--
+-- job_details              details of job instance being created
+-- job_details_v1           <deprecated>
+-- job_service_management   used for clustering and to check lead instance
+--
+
+CREATE TYPE job_status AS ENUM (
+    'ERROR',
+    'EXECUTED',
+    'SCHEDULED',
+    'RETRY',
+    'CANCELED'
+);
+
+CREATE TYPE job_type AS ENUM (
+    'HTTP'
+);
+
+CREATE TABLE job_details (
+    id character varying(50) NOT NULL,
+    correlation_id character varying(50),
+    status character varying(40),
+    last_update timestamp with time zone,
+    retries integer,
+    execution_counter integer,
+    scheduled_id character varying(40),
+    priority integer,
+    recipient jsonb,
+    trigger jsonb,
+    fire_time timestamp with time zone,
+    execution_timeout bigint,
+    execution_timeout_unit character varying(40),
+    created timestamp with time zone
+);
+
+CREATE TABLE job_details_v1 (

```

Comment:
@josedee I checked in the code, and it seems to me this table was used in the past just as a temporary table to migrate job_details data in a new format. In addition, I noticed you already marked it as `deprecated`. IINW, the scope of this DDL package is to create a new database from scratch, so I guess this `v1` is not required and we can remove it from the script.

@porcelli @pefernan @elguardian Can you please confirm ^^ ?

Reply:
good catch, I don't think we are using this table anymore. I think we can just get rid of this in the ddl. Actually we should also remove it in upstream in a flyway script

Reply:
Done